mmda/types/document.py

```diff
@@ -245,16 +244,23 @@ def from_json(cls, doc_dict: Dict) -> "Document":
             )
 
         # 2) convert span group dicts to span gropus
-        field_name_to_span_groups = {}
-        for field_name, span_group_dicts in doc_dict.items():
-            if field_name not in doc.SPECIAL_FIELDS:
+        field_to_span_groups = {}
+        for field, span_group_dicts in doc_dict.items():
+            if field not in doc.SPECIAL_FIELDS:
                 span_groups = [
                     SpanGroup.from_json(span_group_dict=span_group_dict)
                     for span_group_dict in span_group_dicts
                 ]
-                field_name_to_span_groups[field_name] = span_groups
+                field_to_span_groups[field] = span_groups
 
         # 3) load annotations for each field
-        doc.annotate(**field_name_to_span_groups)
+        doc.annotate(**field_to_span_groups)
 
         return doc
+
+    def locate_annotation(self, name: AnnotationName) -> Annotation:
```

Comment:
I am not quite sure on why we need `AnnotationName`. If a name is just a combo of relation name + id, can we just expect a tuple of the two here? or require two args? Overall, having a class to just hold annotation names would just result in computational overhead.

Reply:
see comment below